### DISCUSSION TITLE
CORS policy error

### DISCUSSION THREAD

**Discussion by Navino16** [firefly-iii/firefly-iii#9299] [Firefly III]
Hi,

When running Firefly behind Authentik with `AUTHENTICATION_GUARD=remote_user_guard` I have the following CORS policy error in the web console.

![image](https://github.com/user-attachments/assets/8d4959c8-b9fb-4f53-b103-e1e936d747a5)

Any idea ?

**Comment by JC5** [comment]
It looks like Authentik is trying to inject stuff into Firefly III, right?

**Comment by Navino16** [comment]
I don't know. That's the first time I have this issue. All my other apps that use authentik does work perfectly

**Reply by JC5** [reply]
Sorry, I'm not sure what it could be. The code your browser seems to try to load isn't from Firefly III, that's for sure.
